marquez/client.py

```diff
@@ -0,0 +1,236 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import collections
+import os
+import urllib
+
+import requests
+from marquez.constants import JobRunState
+from marquez.utils import (InvalidRequestException, _compose_path,
+                           handle_response)
+
+
+class MarquezClient(object):
```

Comment:
_Minor_: We could simply define our client class as `Client` allowing for:

```python
import marquez

client = marquez.Client(..)
```

Reply:
Fixed